website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -70,27 +70,28 @@ By supplying the necessary attributes under a `vcs-repository` object, you can c
+| `data.attributes.vcs-repo.oauth-token-id`     | string  |           | The VCS Connection (OAuth Connection + Token) to use. This ID can be obtained from the [oauth-tokens](/terraform/cloud-docs/api-docs/oauth-tokens) endpoint.                                                                                                                                                                                                                                                                                                                                                    |
 | `data.attributes.vcs-repo.tags-regex`         | string  | (nothing) | A regular expression used to match Git tags. Terraform Cloud triggers a run when this value is present and a VCS event occurs that contains a matching Git tag for the regular expression.                                                                                                                                                                                                                                                                                                                      |
 | `data.attributes.vcs-repo`                    | object  | (nothing) | Settings for the workspace's VCS repository. If omitted, the workspace is created without a VCS repo. If included, you must specify at least the `oauth-token-id` and `identifier` keys below.                                                                                                                                                                                                                                                                                                                  |
 | `data.attributes.working-directory`           | string  | (nothing) | A relative path that Terraform will execute within. This defaults to the root of your repository and is typically set to a subdirectory matching the environment when multiple environments exist within the same repository.                                                                                                                                                                                                                                                                                   |
+| `data.attributes.setting-overwrites`          | object  |           | Attributes that have organization-level defaults are defined as keys in this object. Each attribute key stores a boolean value that relates to whether or not they are overwriting the default. In order to set one of these attributes to use the inherited default, set it to false. For example, to set `execution-mode` as the organization default: `setting-overwrites.execution-mode = false`. The setting overwrites object is not necessary to include when overwriting the default.                   |
```

Comment:
I tried shortening this a bit for brevity, I think the points still remain the same but please feel free to push back if you disagree!
```suggestion
| `data.attributes.setting-overwrites`          | object  |           | The keys in this object are attributes that have organization-level defaults. Each attribute key stores a boolean value that relates to whether or not it is overwriting a default. To use the default inherited value, set an attribute's key to false. For example, to set `execution-mode` as the organization default, you set `setting-overwrites.execution-mode = false`. You do not need to include this object when overwriting a default.                   |
```

Reply:
> The setting overwrites object is not necessary to include when overwriting the default.  

IMO we should omit this information for simplicity. Tell the user if you want to inherit, set the key to false; if you want to overwrite set the key to true.

It happens that they dont need to set the key to true in order to overwrite, for backward compatibility....but I think when we are describing the API it makes the most sense to do so in the simplest way possible.

Alternatively, we could say "The default value for each key is `true`"

What do you think?

Reply:
I like that! I tried rephrasing with this context in mind:
```suggestion
| `data.attributes.setting-overwrites`          | object  |           |  The keys in this object are attributes that have organization-level defaults. Each attribute key stores a boolean value which is `true` by default. To overwrite the default inherited value, set an attribute's value to `false`. For example, to set `execution-mode` as the organization default, you set `setting-overwrites.execution-mode = false`.                       |
```